xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/resources/document-formats.js

```diff
@@ -19,196 +19,666 @@
  */
 [
   {
-    "name": "Portable Document Format",
-    "extensions": ["pdf"],
-    "mediaType": "application/pdf",
-    "inputFamily": "DRAWING",
+    "name": "OpenDocument Text",
+    "extensions": [
+      "odt"
+    ],
+    "mediaType": "application/vnd.oasis.opendocument.text",
+    "inputFamily": "TEXT",
     "storeProperties": {
-      "DRAWING": {"FilterName": "draw_pdf_Export"},
-      "SPREADSHEET": {"FilterName": "calc_pdf_Export"},
-      "PRESENTATION": {"FilterName": "impress_pdf_Export"},
-      "TEXT": {"FilterName": "writer_pdf_Export"}
+      "TEXT": {
+        "FilterName": "writer8"
+      },
+      "WEB": {
+        "FilterName": "writerweb8_writer"
+      }
     }
   },
+  // New one
```

Comment:
All the format flagged with "new one" were not present in previous version. I'm not sure we need it, but I don't think it's harmful to have them and it will be easier for upgrades to have them.

Reply:
I don't see any reason for not having everything that is supported in standard jodconverter. I'd drop the "New one" comment as I find it misleading / confusing. We should add comments only for what we customize compared to the standard document formats file.

Reply:
> I'd drop the "New one" comment as I find it misleading / confusing. 

Of course, I added them just to flag them as the diff is not making obvious the changes.